README.md

```diff
@@ -743,6 +744,20 @@ changeset.validate().then(() => {
 
 **[⬆️ back to top](#api)**
 
+#### `afterRollback`
+
+This event is triggered after a rollback of the changeset.
```

Comment:
@bgentry Would you mind adding a few sentences detailing the "use case" for users?  To me it isn't immediately obvious, but I trust that there is a valid use case.